### DISCUSSION TITLE
Standards: EDA Software

### DISCUSSION THREAD

**Discussion by mpadge** [ropensci/statistical-software-review-book#47] [Standards]
## Standards for Exploratory Data Analysis (EDA) Software

This is a place to discuss our [*Standards for Exploratory Data Analysis (EDA) Software*](https://stats-devguide.ropensci.org/standards.html#exploratory-data-analysis). Our standards are community-led, and are intended to evolve in response to ongoing community engagement and feedback. Any suggestions for improvements, modifications, or edits small and large are welcome :clap: 

Participation in these discussions are guided by our [Code of Conduct](https://ropensci.org/code-of-conduct/).

**Comment by grwhumphries** [comment]
@mpadge in section 6.3.2.2 - is there any clarification available on what is meant by "mutli-tabular input"?
